packages/variable-editor/src/components/variables/master/VariablesMaster.tsx

```diff
@@ -88,14 +88,23 @@ export const VariablesMaster = ({ variables, setVariables, setSelectedVariablePa
     setSelectedVariablePath([]);
   };
 
+  /* workaround for "table.getIsSomeRowsSelected" as it returns false if only last remaining row is selected */
+  const isRowSelected = () => document.querySelectorAll('[data-state="selected"]').length > 0;
```

Comment:
I think you can do it easier with something like `Object.keys(table.getState().rowSelection).length > 0`

Reply:
I also think you should move it to `utils/table/table.ts` as it is a general table functionality and can potentially be reused. Then, you can also test it :wink:

Reply:
:+1: was not aware this function exists